### ISSUE TITLE
Settings for training and inference of LightningDiT-B/LightningDiT-L models

### ISSUE THREAD

**Issue by LukashevichIlya** [hustvl/LightningDiT#20]
Hello, I have some questions about training and sampling for the smaller versions of the proposed models (B and L). I haven't been able to clarify some of parameters from the article itself or from configs you provided.

1. Do the recommended training hyperparameters for B or L models differ from XL's ones according to your experience?
2. What CFG scale is preferred for sampling of these models? I have seen your comment in [config](https://github.com/hustvl/LightningDiT/blob/main/configs/config_details.yaml) about different CFG scales for XL model trained for 800 and 64 epochs to obtain paper's results, therefore, I wonder if there is some difference for other model versions. Additionally, do you recommend to use CFG interval and timestep shift in this case?

**Comment by JingfengYao** [comment]
Hi, thanks for the question. Firstly, from a training perspective, we utilized identical hyperparameters for both the -B and -XL models. Regarding CFG, our experience suggests that models with better conditional generation capabilities require a smaller CFG scale. Based on our experiments, it appears that employing interval and timestep shifting is effective if one is aiming for a higher FID.

**Comment by LukashevichIlya** [comment]
Thanks for details and suggestions! I found that upscaling LR for smaller models helped in optimization of the loss. As for sampling, as I understand, hyperparameters highly depend on what we are aiming for (image quality, FID, etc.).
